src/content/docs/developer/modules/other/external-qa-checks.mdx

```diff
@@ -0,0 +1,246 @@
+---
+title: External QA Check Module
+description: Add an external QA check for advanced AI-powered verifications.
+slug: developer/crowdin-apps-module-external-qa-check
+sidebar:
+  label: External QA Check
+---
+
+This module allows for the integration of advanced AI-powered QA checks, enabling verification of translations for nuanced issues that cannot be detected by [default QA checks](/enterprise/project-settings/qa-checks/) or JavaScript-based [Custom QA checks](/enterprise/custom-qa-checks/#creating-and-adding-custom-qa-checks).
+
+## Access
+
+You can grant access to this module to one of the following user categories:
+
+For Crowdin:
+
+* Only me (i.e., project owner)
+* All project members
+* Selected users
+
+For Crowdin Enterprise:
+
+* Only organization admins
+* All users in the organization projects
+* Selected users
+
+## Structure
+
+```json title="manifest.json"
+{
+  "modules": {
+    "external-qa-check": [
+      {
+        "key": "custom-check-qa",
+        "name": "QA Check",
+        "description": "Description",
+        "runQaCheckUrl": "/validate",
+        "getBatchSizeUrl": "/batch-size",
+        "url": "/settings/index.html"
+      }
+    ]
+  }
+}
+```
+
+## Properties
+
+<table>
+  <tbody>
+    <tr>
+      <td><code>key</code></td>
+      <td>
+        <p><strong>Type:</strong> <code>string</code></p>
+        <p><strong>Required:</strong> yes</p>
+        <p><strong>Description:</strong> Module identifier within the Crowdin app.</p>
+      </td>
+    </tr>
+    <tr>
+      <td><code>name</code></td>
+      <td>
+        <p><strong>Type:</strong> <code>string</code></p>
+        <p><strong>Required:</strong> yes</p>
+        <p><strong>Description:</strong> The human-readable name of the module.</p>
+      </td>
+    </tr>
+    <tr>
+      <td><code>description</code></td>
+      <td>
+        <p><strong>Type:</strong> <code>string</code></p>
+        <p><strong>Description:</strong> The human-readable description of what the module does.<br/>
+         The description will be visible in the Crowdin Enterprise UI.</p>
```

Comment:
```suggestion
         The description will be visible in the Crowdin UI.</p>
```

Reply:
Considering that this module is only available in Enterprise, I'd suggest keeping the current version